hrm-service/service-tests/contact-job/jobTypes/retrieve-transcript.test.ts

```diff
@@ -162,7 +166,7 @@ describe('publish retrieve-transcript job type', () => {
 
     expect(updatedRetrieveContactTranscriptJob.completed).toBeNull();
     expect(
-      isAfter(updatedRetrieveContactTranscriptJob.lastAttempt!, startedTimestamp),
+      moment(updatedRetrieveContactTranscriptJob.lastAttempt!).isAfter(startedTimestamp),
```

Comment:
We tend to prefer `date-fns` to `moment` in other places (I think) - what's the thinking behind changing?

Reply:
Tests were failing when using Date-fns.after because it wasn't working for microsecond differences.

Reply:
But I just tried again after reverting this change. and `date-fns.isAfter()` appears to work with milliseconds now. Strange. I pushed the change to move back to date-fns to this PR.